### PR TITLE
Azure Cloud Cost: refactoring + add budget as add. info. + add. info renaming

### PR DESCRIPTION
**Pull request by mbenguig**

### CODE REVIEW

File: infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java [hunk 1]
```diff
@@ -193,130 +169,131 @@
...
+        this.additionalProperties = getParameter(parameters, Indexes.ADDITIONAL_PROPERTIES.index);
+        this.resourceUsageRefreshFreqInMin = Integer.parseInt(getParameter(parameters,
+                                                                           Indexes.BILLING_RESOURCE_USAGE_REFRESH_FREQ_IN_MIN.index));
+        this.rateCardRefreshFreqInMin = Integer.parseInt(getParameter(parameters,
+                                                                      Indexes.BILLING_RATE_CARD_REFRESH_FREQ_IN_MIN.index));
+        this.offerId = getParameter(parameters, Indexes.BILLING_OFFER_ID.index);
+        this.currency = getParameter(parameters, Indexes.BILLING_CURRENCY.index);
+        this.locale = getParameter(parameters, Indexes.BILLING_LOCALE.index);
+        this.regionInfo = getParameter(parameters, Indexes.BILLING_REGION_INFO.index);
+        this.maxBudget = Double.parseDouble(getParameter(parameters, Indexes.BILLING_BUDGET.index));
```
Comment (mboussaa) [inline infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java]: no need to catch exceptions with ur `parseInt` and `parseDouble` operations?

Comment (mbenguig) [reply infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java]: You re right but I strictly followed the existing behavior.. Exemple in the same function  "this.numberOfInstances = Integer.parseInt(getParameter(parameters, Indexes.NUMBER_OF_INSTANCES.index));"

Comment (mboussaa) [reply infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java]: Ok

Comment (mboussaa) [reply infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java]: Ok

**Pull request merged**